containers/containers.go

```diff
@@ -27,7 +29,10 @@ type RuntimeInfo struct {
 
 type Store interface {
 	Get(ctx context.Context, id string) (Container, error)
-	List(ctx context.Context, filter string) ([]Container, error)
+
+	// List returns containers that match one or more of the provided filters.
```

Comment:
Then you use the syntax to represent `AND`. For example, `foo=bar,labels.boo=true` must match both labels.boo and foo.